testing/src/main/java/io/stargate/it/http/docsapi/BaseDocumentApiV2Test.java

```diff
@@ -163,54 +137,54 @@ public void testUnauthorized() throws IOException {
 
     // Missing token header
     RestUtils.post(
-        null, hostWithPort + "/v2/namespaces/" + keyspace + "/collections/collection", data, 401);
+        null, hostWithPort + "/v2/namespaces/" + KEYSPACE + "/collections/collection", data, 401);
```

Comment:
Restv2ApiTest uses String.format as below:

String.format("%s/v2/namespaces/%s/collections/collection", hostWithPort, KEYSPACE);
We can even extract %s/v2/namespaces/%s/collections/collection as a constant.

Reply:
Yup, will do..